capstone/src/main/webapp/text_analyzer.html

```diff
@@ -51,40 +51,68 @@ <h1>Text Analysis</h1>
             <textarea type="text" class="form-control" id="textForAnalysis" aria-describedby="inputText" placeholder="Input Text To Be Analyzed Here" rows="3"></textarea>
             <small id="disclaimer" class="form-text text-muted">Data may be stored for QA/QC purposes.</small>
           </div>
-          <select id="languageForAnalysis" class="form-control" onchange="showAvailableAttributes()">
-            <option value="en">English</option>
-            <option value="es">Spanish</option>
-            <option value="fr">French</option>
-            <option value="de">German</option>
-            <option value="it">Italian</option>
-            <option value="pt">Portuguese</option>
-          </select>
-          <div class="dropdown" id="attribute-dropdown">
-            <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown">Attributes</button>
-            <ul class="dropdown-menu checkbox-menu allow-focus" id="available-attributes"></ul>
-          </div>
-          <div class="dropdown" id="analysis-dropdown">
-            <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown">Analysis Type</button>
-            <ul class="dropdown-menu checkbox-menu allow-focus" id="available-attributes">
-              <li class="active">
-                <label class="form-check-label" for="bySentence">
-                  <input class="form-check-input" type="radio" name="analysisRadios" id="bySentence" value="SENTENCE" checked>
-                  By sentence
-                </label>
-              </li>
-              <li>
-                <label class="form-check-label" for="byWord">
-                  <input class="form-check-input" type="radio" name="analysisRadios" id="byWord" value="WORD">
-                  By word
-                </label>
-              </li>
-              <li>
-                <label class="form-check-label" for="noAnalysis">
-                  <input class="form-check-input" type="radio" name="analysisRadios" id="noAnalysis" value="NONE">
-                  No analysis
-                </label>
-              </li>
-            </ul>
+          <div class="row">
+            <div class="col-12 justify-content-center form-inline">
+              <div class="dropdown" id="language-dropdown" onchange="showAvailableAttributes()">
+                <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown" id="language-button">Language: EN</button>
+                <ul class="dropdown-menu checkbox-menu allow-focus" id="available-languages">
+                  <li class="active">
```

Comment:
Would it be more elegant to generate this list from JS? This would also make it easier to update the website when new languages are added

Reply:
I agree - it was just easier to hard-code it at first. I'll do this in a separate branch.

Reply:
Are we sticking to generating lists in JS or transitioning to generating them in HTML?

Reply:
Without using something like Angular, I think JS is the way to do this.